src/sentry/tasks/statistical_detectors.py

```diff
@@ -306,6 +306,10 @@ def _detect_transaction_change_points(
             yield from detect_breakpoints(request)["data"]
         except Exception as e:
             sentry_sdk.capture_exception(e)
+            metrics.incr(
+                "statistical_detectors.breakpoint.errors",
+                tags={"type": "transactions", "error": str(e)},
```

Comment:
I dont think we'd want to tag the metric with the error because it could be high cardinality.